Berksfile

```diff
@@ -5,3 +5,4 @@ cookbook 'apt'
 cookbook 'os_prepare', path: './test/cookbooks/os_prepare'
 cookbook 'runit', github: 'hw-cookbooks/runit'
 cookbook 'ssh-hardening', git: 'https://github.com/dev-sec/chef-ssh-hardening.git'
+cookbook 'openssl', '~> 6'
```

Comment:
Switch to openssl cookbook 7.x since https://github.com/chef-cookbooks/openssl/issues/54 is solved now